code/implementation/register/src/main/java/io/cattle/platform/register/process/RegisterTokenAccountCreate.java

```diff
@@ -1,66 +0,0 @@
-package io.cattle.platform.register.process;
```

Comment:
We determine this entire class was unneeded legacy code. Was randomly creating registration tokens that should never be used.